app/controllers/application_controller.rb

```diff
@@ -9,4 +11,15 @@ class ApplicationController < ActionController::Base
       password: ENV.fetch("BASIC_AUTH_PASSWORD"),
     )
   end
+
+protected
+
+  def error_403(exception); error(403, exception); end
+
+  def error(status_code, exception = nil)
+    if exception && defined? GovukError
+      GovukError.notify exception
+    end
```

Comment:
I don't think we want to inform sentry when we get a client error from ContentStore. It doesn't seem something that a developer would action.

I appreciate this is lifted and shifted from calendars though I'd question the relevance of it there too.

Reply:
Makes sense 👍 removed.